app/Services/Post.php

```diff
@@ -154,6 +155,15 @@ public static function getPostFromUrl( $postUrl ) {
 		// Create the ACF Widgets from DOM content
 		$widgets = Widget::getWidgetsFromDom( $postDom );
 
+		// if there is a video header, convert to embed widget
+		if ($hero == "no-media") {
+			$headerEmbed = Video::getVideoFromHeader($postObject->provider, $postObject->videoId);
+			array_unshift($widgets, $headerEmbed);
+		}
+
+		var_dump($widgets);
```

Comment:
- this was very stupid and not supposed to be in here at all, especially the `die` after it!